schema-loader/src/integration-test/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -0,0 +1,180 @@
+        CONFIG_FILE,
+        "--schema-file",
+        SCHEMA_FILE,
+        "--coordinator");
+  }
+
+  protected List<String> getSchemaLoaderDeletionCommandArgs() {
+    List<String> args = new ArrayList<>(getSchemaLoaderCreationCommandArgs());
+    args.add("-D");
+    return args;
+  }
+
+  @Test
+  public void storageSpecificCommand_GivenProperArgs_ShouldCreateTablesThenDeleteTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldCreateTables();
+    deleteTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldDeleteTables();
+  }
+
+  @Test
+  public void schemaLoaderCommand_GivenProperArgs_ShouldCreateTablesThenDeleteTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldCreateTables();
+    deleteTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldDeleteTables();
+  }
+
+  public void createTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldCreateTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndCommandArgs_ShouldCreateTables(
+        getStorageSpecificCreationCommandArgs());
+  }
+
+  public void createTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldCreateTables()
+      throws Exception {
+    createTables_GivenProperSchemaFileAndCommandArgs_ShouldCreateTables(
+        getSchemaLoaderCreationCommandArgs());
+  }
+
+  private void createTables_GivenProperSchemaFileAndCommandArgs_ShouldCreateTables(
+      List<String> args) throws Exception {
+    // Act
+    int exitCode = executeCommandWithArgs(args);
+
+    // Assert
+    assertTableCreation(exitCode);
+  }
+
+  public void deleteTables_GivenProperSchemaFileAndStorageSpecificCommandArgs_ShouldDeleteTables()
+      throws Exception {
+    deleteTables_GivenProperSchemaFileAndCommandArgs_ShouldDeleteTables(
+        getStorageSpecificDeletionCommandArgs());
+  }
+
+  public void deleteTables_GivenProperSchemaFileAndSchemaLoaderCommandArgs_ShouldDeleteTables()
+      throws Exception {
+    deleteTables_GivenProperSchemaFileAndCommandArgs_ShouldDeleteTables(
+        getSchemaLoaderDeletionCommandArgs());
+  }
+
+  private void deleteTables_GivenProperSchemaFileAndCommandArgs_ShouldDeleteTables(
+      List<String> args) throws Exception {
+    // Act
+    int exitCode = executeCommandWithArgs(args);
+
+    // Assert
+    assertTableDeletion(exitCode);
+  }
+
+  private int executeCommandWithArgs(List<String> args) throws Exception {
+    // Arrange
+    ProcessBuilder processBuilder = new ProcessBuilder(args);
+
+    // Act
+    Process process = processBuilder.start();
+    try {
+      try (final BufferedReader input =
+          new BufferedReader(
+              new InputStreamReader(process.getErrorStream(), Charset.defaultCharset()))) {
+        String line;
+        while ((line = input.readLine()) != null) {
+          System.out.println(line);
+        }
+      }
+    } catch (Exception e) {
```

Comment:
BTW, do we really need to catch `Exception` here?

Reply:
I removed it [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/aa0a087c02f328990bf34f07275bbc3730461a9f)